mesop/runtime/runtime.py

```diff
@@ -54,8 +55,15 @@ def __init__(self):
     self._state_classes: list[type[Any]] = []
     self._loading_errors: list[pb.ServerError] = []
     self._has_served_traffic = False
+    self._contexts = {}
 
   def context(self) -> Context:
+    if MESOP_WEBSOCKETS_ENABLED and hasattr(request, "sid"):
+      # flask-socketio adds sid (session id) to the request object.
+      sid = request.sid  # type: ignore
+      if sid not in self._contexts:
+        self._contexts[sid] = self.create_context()
```

Comment:
[question] Is runtime created every request or is it a singleton? If it's the latter, I was wondering if _contexts would need to be cleared